base/system/services/rpcserver.c

```diff
@@ -779,7 +779,7 @@ ScmCanonDriverImagePath(DWORD dwStartType,
 
 /* Internal recursive function */
```

Comment:
you stripped the static, so it is no longer an internal func.
```suggestion
/* Recursive function */
```

Reply:
Changed. Thanks.